icons/angular/angular-original.svg

```diff
@@ -1,8 +1,39 @@
+dy5k+HvkAsQgaAFBuJWJ41pmBuxlAu4xVDQBbF16HGaSNjaKo9jA0uKEZ2+0Lj8yG+Ed9JYksl5GjKIIs3WJ2XLFbF0yYBQ3xHTxyRGDtfowx0pxjWniaDkiq8UYD8XeXSe1VSpgJ8VgtNsDR4SPAqIBAwMDAAAA//8aLUipBEAF6g+fQiILURxdejwTRxitKoJjs7jUEhAna5IJS0GKo0sPV4M20cSAVkgi6hRUdagFKUwt9oKUAU+hiqsgJXd8FIXNiK1Lj3+sFFv3HtckEWYXH99YKaYcv70Ug2KtEYOAnSSOyB4FJAEGBgYAAAAA//8a7dpTCTDb6DNwf9jHwFaRgGYgCV16HIUY/q4pgUKUCG2kqSWtW00X84hWj6VXgGUUFh9ALcZxDH/gXY6G2U1nxCLHiCXesXXTEWKomJEBfesoRJ9CrRGD4W7v0UKUmoCBgQEAAAD//xptkdIA/DtygeF3B6i7fx5n9xtbZ5joLj2KWtSClKguPS5xotUy4hdHaYRjaZXiOLzkP4q56C1QBiwTRwxoS55wdf8xW6LIerC1Tknv1qN3n/HI4ZxUwtFiReneM+A8Bg9rqxSql89ehkFvtz+OCB4FFAEGBgYAAAAA//8aLUhpCEAF6g/vAjQLcIyNYo0FbGrxTOIQW5CSUrjiFMecrUcuYHFOycALPmQ5fAUpqjoMPs7ZegaMwpFwQcpAZKGJXQ7b+CWqXtxLo3CdWYo5HoppDq6zQ0F6+exlGWTrzBj47aRxRO4ooBgwMDAAAAAA//8aLUjpAH53LGD4d/gCpIXKgON6DiyFKyNWcZRBSBQxUsZcsQJs45p4RxWwDSIwonmJ+IKUAeZrpNYsvoL0PwMjyqJ+7AUpA1oBimucFLs46S1TND4jZmsU55gpvqVO8FYpemGJfTkVL6gArbUAt0RHAY0BAwMDAAAA//8aLUjpCMBd/vb54EIVAbAVpNAsTWwhilUtnmP1CBaujPjVoohjKUxRCkt087BNLiHUEDolH4WPMsmEvyDFX2Ci83EVmgx4C0VcXXx8k03orVKMVihGYYlnCADchZdjkK61Gi1A6QkYGBgAAAAA//8aLUgHACAKVEgLlbQDemK8AAAFaElEQVTWKBZBEsZcKe++o6vH1oJlRNOGXjBjH/9kYMC/JRR/QcqAUTDiYuMrHBHqmMjq6mMtLBkx1ePs6mPM+KPxsV0JwsDIwAMqQOtswC3RUUBnwMDAANBoQTqAAFygtiEVqGCApzWKo5VJ3FABA+niuNaOYqjHLCixFqRohS9m15+R+IL0P6Y89vFPmDwpLVFMcULjo7jZ6C1Nwi1YXOOg2ApbyTo7Bqk6G9zxNApoDxgYGAAAAAD//xotSAcBAHX1ES1UHBNMeMY7kTvL2Jjo6okSY0BtPRI0A6WwJLMgJbC3HsVcnLP1xBSu2AtM7GqIbY0yYC0AGZDGPmHm4+7eM2I58g5dPaRglqizBxeio2AQAAYGBgAAAAD//xotSAcRABWkf0AF6qHzaI7C3cok5yAT0sSJKUgxC0rqFqQMaAUdekHKgFSoobJJKUixj6fiGwLA171HLVQxu/fYzi5lxLgnCaUAZWBkEK93BBeio2AQAQYGBgAAAAD//xotSAchABWooFbqn7Z5UMfhWayPYwYfKyBbHNs4KIZLSCpIsctTvyDFPhbKgKXAZMAoENHFyWmtou9OIq17j5ATq3diEK9zxBGBo2BAAQMDAwAAAP//Gi1IBzkAFabgghWjlYqjEMUhhFOcKLWkFqToE0no8lgKUixjnjgLUqwTVqS0SBmIKEjR+fjHSXF31VELSKwFJ3r3HapOpN6FgdteiYHbXhFHJI2CQQEYGBgAAAAA//8aLUiHCIC0Us8z/GmdB3cwcQeZUEuckYBZBApSLOeSohSS/9H1YRaUDCQVpAxIhSU2eUYUMfx8UsdMGZDEsHfvcY2ncjqoMIjWuzJw2SthC+RRMBgBAwMDAAAA//8aLUiHIAC1Uv8fOoe1lUrzghRnaqF3QcqAs8Akho2/S4+rIMW2jpS0ZVKYJ0MxgQtPTnsVBpF6N1yBOwoGM2BgYAAAAAD//xotSIc4+NsK6vojFarU6NbjlMO3rhS1oKRfQYq/FcqApbVJapeeUAFLihhy916o3pNBqH70GLshDxgYGAAAAAD//xotSIcRABeq2Fqq1GqlDoKCFCyGZVsoMQUpA5ZCFRcffyuU/IJUsMGbgcNBlYHTXhVXII+CoQYYGBgAAAAA//8aLUiHKUApVKna3UcvEGFyeApSLPpQC1ri99cz/EctXDHV4i48GVAKRQaUApO41irmBBXy+lBchSh/gy8Dh70aA4eDGo6AHQVDGjAwMAAAAAD//xotSEcAABWm/w+dx2yt0rQgZUAp/BiQ+XgKUqwtUGQxPAUpcoFJeJIJO5/wOCm2CSVUmt1BA4z5631xBOQoGFaAgYEBAAAA//8aLUhHIIAXrAdJGAZgwF4g4hP/T9eCFHUoALMVyoC1kCR9nBS7PG9DIAO7PagAVccTgKNgWAIGBgYAAAAA//8aLUhHAWrBehDLSgAGPIUoHjnKClIGrAUhsV17/C1S0gpSRDcewmZ10GJgc9BiYLfXZGBz0BhNQCMdMDAwAAAAAP//Gi1IRwEGABesB6FDAQfRVwPgKkgxxVFn2BnoVJAyYBSQDCjddEx5QhNQ3A2hYD53ffBoYhkFmICBgQEAAAD//xotSEcB0eBvM2IzAKiARS1ksRWk6GOnOApSQmeUklWQ4mt1MmAtODkbwuFyXPWhowljFBAHGBgYAAAAAP//Gi1IRwFVwN+m+XBjkAtZ9IOaYYAWBSnq+CqmHIuDLhjD5FjtdRhYHHRGE8AooAwwMDAAAAAA//8aLUhHAd3An6YF2K1CO1H/38GLDH8PXMAoINnq4zH36CMbA+Wz2OsxMIMLzFEwCugAGBgYAAAAAP//AwB1danmftkU3gAAAABJRU5ErkJggg=="/>
```

Comment:
same here.

Reply:
I found this on their website:

```svg
<svg _ngcontent-ng-c3811030075="" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 223 236" width="32"><g _ngcontent-ng-c3811030075="" clip-path="url(#a)"><path _ngcontent-ng-c3811030075="" fill="url(#b)" d="m222.077 39.192-8.019 125.923L137.387 0l84.69 39.192Zm-53.105 162.825-57.933 33.056-57.934-33.056 11.783-28.556h92.301l11.783 28.556ZM111.039 62.675l30.357 73.803H80.681l30.358-73.803ZM7.937 165.115 0 39.192 84.69 0 7.937 165.115Z"></path><path _ngcontent-ng-c3811030075="" fill="url(#c)" d="m222.077 39.192-8.019 125.923L137.387 0l84.69 39.192Zm-53.105 162.825-57.933 33.056-57.934-33.056 11.783-28.556h92.301l11.783 28.556ZM111.039 62.675l30.357 73.803H80.681l30.358-73.803ZM7.937 165.115 0 39.192 84.69 0 7.937 165.115Z"></path></g><defs _ngcontent-ng-c3811030075=""><linearGradient _ngcontent-ng-c3811030075="" id="b" x1="49.009" x2="225.829" y1="213.75" y2="129.722" gradientUnits="userSpaceOnUse"><stop _ngcontent-ng-c3811030075="" stop-color="#E40035"></stop><stop _ngcontent-ng-c3811030075="" offset=".24" stop-color="#F60A48"></stop><stop _ngcontent-ng-c3811030075="" offset=".352" stop-color="#F20755"></stop><stop _ngcontent-ng-c3811030075="" offset=".494" stop-color="#DC087D"></stop><stop _ngcontent-ng-c3811030075="" offset=".745" stop-color="#9717E7"></stop><stop _ngcontent-ng-c3811030075="" offset="1" stop-color="#6C00F5"></stop></linearGradient><linearGradient _ngcontent-ng-c3811030075="" id="c" x1="41.025" x2="156.741" y1="28.344" y2="160.344" gradientUnits="userSpaceOnUse"><stop _ngcontent-ng-c3811030075="" stop-color="#FF31D9"></stop><stop _ngcontent-ng-c3811030075="" offset="1" stop-color="#FF5BE1" stop-opacity="0"></stop></linearGradient><clipPath _ngcontent-ng-c3811030075="" id="a"><path _ngcontent-ng-c3811030075="" fill="#fff" d="M0 0h223v236H0z"></path></clipPath></defs></svg>
```
Just need to optimize, resize and center it then it should be good to go